style.css

```diff
@@ -0,0 +1,118 @@
+header {
+  position: absolute;
+  width: 423px;
```

Comment:
- [ Optional ] It is not good practice to use fixed-sized width for it makes your website not responsive.